shared/tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "exclude": ["node_modules"],
   "compilerOptions": {
     "target": "ESNext",
-    "moduleResolution": "NodeNext",
+    "moduleResolution": "Node16",
```

Comment:
With NodeNext it is not working properly?

Reply:
Yes, eslint will automatically consider this package as esm. So we can either remove type module, or change module resolution